### ISSUE TITLE
Cannot add empty string property in vertex

### ISSUE THREAD

**Issue by amit12cool** [zposten/gremlin-cosmos#22]
Azure cosmos gremlin supports adding empty string property but this package doesn't.

Code:- 
```
    const query = Query.g.addV('test');
    query.property('env', '');
```

The above code converts to:-

```
"g.addV('test').property('env)"
```
On firing above query Azure gives an error:-

```
Gremlin Query Compilation Error: Unable to bind to method 'property', with arguments of type: (String) @ line 1, column 1018.
```

The correct query made by this package should be like below:-

```
"g.addV("test").property("env", "")"
```
Please fix this issue

**Comment by amit12cool** [comment]
@zposten  I know the cause of this issue is in ```quoteAndCombine``` function. The filter function removes the empty string and returns only the valid strings. If you want, I can raise a PR for same.

**Comment by amit12cool** [comment]
@zposten any updates?

**Comment by zposten** [comment]
@amit12cool I would welcome your PR for this change!  Please also add unit tests around this case so that it doesn't pop up in the future

**Comment by amit12cool** [comment]
@zposten I have raised a PR for above change here https://github.com/zposten/gremlin-cosmos/pull/27

**Comment by zposten** [comment]
Fixed by #27 and published in `v0.0.13`
